### DISCUSSION TITLE
Gas fee so high

### DISCUSSION THREAD

**Discussion by jj56313751** [Cyfrin/foundry-full-course-cu#2559] [General]
In this course [Deploying a smart contract on testnet (Sepolia)](https://updraft.cyfrin.io/courses/foundry/foundry-simple-storage/deploying-smart-contract-testnet-sepolia)
When I execute the following command, the deployment failed

```bash
forge script script/DeploySimpleStorage.s.sol --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --broadcast
```
and I got an error

```bash
== Return ==
0: contract SimpleStorage 0x738fd774Bc4a3Af8920734D97D0a4F34Bc86b375

## Setting up 1 EVM.

==========================

Chain 11155111

Estimated gas price: 450.87871117 gwei

Estimated total gas used for script: 542126

Estimated amount required: 0.24443307217174742 ETH

==========================
Error: 
Failed to send transaction

Context:
- server returned an error response: error code -32000: insufficient funds for gas * price + value: balance 137322203157228822, tx cost 244433072171747420, overshot 107110869014518598
```

I don’t have that much ETH in my wallet, I can’t afford frequent deployments. I can only get 0.01 from faucet everyday.

In the video course, the fee weren’t that high.

![QQ_1727424317792](https://github.com/user-attachments/assets/24fd27ee-170b-416b-a151-56f01c33b62f)

**Comment by EngrPips** [comment]
Hello @jj56313751, Use faucets  https://faucets.chain.link/sepolia  and get more eth

**Reply by jj56313751** [reply]
Thanks, I did get some ETH, but actually it's still not enough 😢

**Reply by EngrPips** [reply]
send your address.

**Reply by jj56313751** [reply]
0x2a42fd00F3e2d19D446960d1F247544D6911b77e

**Reply by EngrPips** [reply]
I sent you some https://sepolia.etherscan.io/tx/0x286bb81ba25195bf7128c022fd7a2c0bc1906c97ee681811d825cd1433d73a06 , please don't tell anyone.

**Reply by EngrPips** [reply]
lol 😂

**Reply by jj56313751** [reply]
Thank you so much😉

**Reply by EngrPips** [reply]
Anytime, Friend.
